### PR TITLE
adapt fileaway.py for Python 3

### PR DESCRIPTION
**Pull request by aperum**
This will make this script work with Python 3. I'm no python programmer but i think the iter change makes this incompatible with Python 2.

### CODE REVIEW

File: python/fileaway.py [hunk 1]
```diff
@@ -178,11 +180,11 @@ def check_timer():
 
 if __name__ == "__main__":
   if w.register(SCRIPT_NAME, SCRIPT_AUTHOR, SCRIPT_VERSION, SCRIPT_LICENSE, SCRIPT_DESC, "", ""):
-    for option, default_value in settings.iteritems():
+    for option, default_value in iter(settings.items()):
```
Comment (flashcode) [inline python/fileaway.py]:
```suggestion
    for option, default_value in settings.items():
```
Using `.items()` is enough here.

File: python/fileaway.py [hunk 2]
```diff
@@ -72,19 +72,21 @@
 #  -Handles improper commands
 # Version 1.0.2 release - Jun 15, 2011
 #  -Added alternative for xset users (credit: sherpa9 at irc.freenode.net)
+# Version 1.0.3 release - Dec 19, 2019
+#  -Adapt for python3
```
Comment (flashcode) [inline python/fileaway.py]: To use the print function in Python 2, you should add this import:

```python
from __future__ import print_function
```

Comment (aperum) [reply python/fileaway.py]: added, tnx

**Comment by flashcode** [comment]
Merged, thanks.

**Pull request closed without merging**